ssh/data/run.sh

```diff
@@ -47,8 +47,15 @@ touch /data/.bash_history
 chmod 600 /data/.bash_history
 ln -s -f /data/.bash_history /root/.bash_history
 
-# Store token for Hass.io API
-echo "export HASSIO_TOKEN=${HASSIO_TOKEN}" >> /root/.bash_profile
+# Persist .bash_profile by redirecting .bash_profile to /data
+if bashio::fs.file_exists /data/.bash_profile; then
+  sed -i "s/export HASSIO_TOKEN=.*/export HASSIO_TOKEN=${HASSIO_TOKEN}/" /data/.bash_profile
+else
+  echo "export HASSIO_TOKEN=${HASSIO_TOKEN}" > /root/.bash_profile
```

Comment:
I just noticed this!

```suggestion
  echo "export HASSIO_TOKEN=${HASSIO_TOKEN}" > /data/.bash_profile
```

Or else the file is never created!

Reply:
Good catch.  Corrected.